orion/sites/admin/src/index.js

```diff
@@ -66,10 +74,10 @@ class App extends React.Component {
           <Route path="/achievements/:Id/edit" component={AchieveEditMatch}/>
           <Route path="/achievements/add" component={AchieveEdit}/>
           <Route path="/achievements" component={Achieve}/>
-          <Route path="/locations/:locId/edit" component={LocationEditMatch}/>
+          <Route path="/locations/:locationId/edit" component={LocationEditMatch}/>
```

Comment:
The current version of locations uses `:locId` now. Do not change to `:locationId`